lib/parser.js

```diff
@@ -275,7 +276,7 @@ Parser.prototype._trimNodes = function (node) {
   }
 };
 
-Parser.prototype.includeFile = function (file, cb) {
+Parser.prototype.includeFile = function (file, cb, config) {
```

Comment:
I am thinking about deprecating the callback for the API and use Promise only, and I don't think we are still using callback anywhere in the codebase, so I think it's safe to change to put `config` after the `file` parameter here (do a check before you do so) :)  
Let's create an issue and get rid of callback in a separate PR.

Reply:
Checked and changed!